index.html

```diff
@@ -6,9 +6,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Bootstrap Playground</title>
   </head>
-  <body>
-    <div class="container">
-      Nothing here yet...
+
```

Comment:
Could you add this changes to an example page, and just link them on `index.html` (`examples/accordions.html`)?